silkworm/common/log_test.cpp

```diff
@@ -54,30 +54,27 @@ namespace {
 bool test_logging()
 {
     // test true branch of macro
-    logger.level(LogCrit);
-    SILKWORM_LOG(LogCrit)  << "LogCrit"  << std::endl;
+    SILKWORM_LOG_TO_LEVEL(logger, LogCrit);
+    SILKWORM_LOG_TO(logger, LogCrit)  << "LogCrit"  << std::endl;
     if (!test_log("CRIT ", infix, "LogCrit"))  return false;
-    SILKWORM_LOG(LogError) << "LogError" << std::endl;
+    SILKWORM_LOG_TO(logger, LogError) << "LogError" << std::endl;
     if (!test_log("ERROR", infix, "LogError")) return false;
-    SILKWORM_LOG(LogWarn)  << "LogWarn"  << std::endl;
+    SILKWORM_LOG_TO(logger, LogWarn)  << "LogWarn"  << std::endl;
     if (!test_log("WARN ", infix, "LogWarn"))  return false;
-    SILKWORM_LOG(LogInfo)  << "LogInfo"  << std::endl;
+    SILKWORM_LOG_TO(logger, LogInfo)  << "LogInfo"  << std::endl;
     if (!test_log("INFO ", infix, "LogInfo"))  return false;
-    SILKWORM_LOG(LogDebug) << "LogDebug" << std::endl;
+    SILKWORM_LOG_TO(logger, LogDebug) << "LogDebug" << std::endl;
     if (!test_log("DEBUG", infix, "LogDebug")) return false;
-    SILKWORM_LOG(LogTrace) << "LogTrace" << std::endl;
+    SILKWORM_LOG_TO(logger, LogTrace) << "LogTrace" << std::endl;
     if (!test_log("TRACE", infix, "LogTrace")) return false;
 
     // test false branch of macro
-    logger.level(LogTrace);
-    SILKWORM_LOG(LogDebug) << "LogDebug" << std::endl;
+    SILKWORM_LOG_TO_LEVEL(logger, LogTrace);
+    SILKWORM_LOG_TO(logger, LogDebug) << "LogDebug" << std::endl;
     if (!test_log("", "", "")) return false;
 
     return true;
 }
 
-TEST_CASE("Logging") {
-    CHECK(test_logging() == true);
```

Comment:
Why did you remove the CHECK? Unit tests should assert, otherwise they are pointless.

Reply:
Accident in testing. Cleaned it all up.